README.md

```diff
@@ -163,6 +163,10 @@ request state:
 * `M-merged`: The PR was successfully merged (and probably closed).
   The bot will not attempt to merge this PR again even if it is
   reopened. The bot never removes this label.
+* `M-abandoned-staging-checks`: The PR state changed, making the
```

Comment:
Just a question: Would this label be added if the PR state remains the same but the state of the `auto` branch changes? For example, if another PR (or a human) force-pushes their own commit to the auto branch? I would like to polish this wording, but I want to make sure I understand what triggers the label.